eng/SharedFramework.Local.props

```diff
@@ -56,6 +56,7 @@
     <AspNetCoreAppReference Include="Microsoft.AspNetCore.Cors" />
     <AspNetCoreAppReference Include="Microsoft.AspNetCore.Diagnostics.Abstractions" />
     <AspNetCoreAppReference Include="Microsoft.AspNetCore.Diagnostics" />
+    <AspNetCoreAppReference Include="Microsoft.AspNetCore.HeaderPropagation" />
```

Comment:
Can you remove this from the shared framework? We're generally very very cautious about what we put in there since it's very difficult to remove things later on. We're trying to keep it to things that every single app will use (moving forward at least). 

We can still ship it as an official package as part of our release, it just wouldn't be included in the shared framework.

@dougbu Do you know if we need to do anything special to make sure this package is in the set of packages that will end up on NuGet? If we remove it from the shared framework, I want to make sure we know it will end up landing on nuget.org

```suggestion
```

Reply:
I don't remember exactly. Removing the `<AspNetCoreAppReference />` item may be sufficient. If not, there should be a project property you can set&hellip;

Reply:
I've removed `IsAspNetCoreApp` ans set `IsShippingPackage` https://github.com/aspnet/AspNetCore/pull/7921/commits/61df6957ab65b9b2bb9e4082cd0d4c2ea1ef767f.
Looking at https://github.com/aspnet/AspNetCore/blob/master/Directory.Build.targets#L5-L17 this should be the correct approach and executing `dotnet pack` locally it create the package in the `Shipping` folder.

Can you please confirm @dougbu ?

Reply:
Seems to be working fine :smiley_cat: And, the CI build logs contain the following which also looks right:
```
  Successfully created package 'F:\workspace\_work\1\s\artifacts\packages\Release\Shipping\Microsoft.AspNetCore.HeaderPropagation.3.0.0-preview4-19177-46.nupkg'.
  Successfully created package 'F:\workspace\_work\1\s\artifacts\packages\Release\Shipping\Microsoft.AspNetCore.HeaderPropagation.3.0.0-preview4-19177-46.symbols.nupkg'.
```

Unfortunately, this is a fork and so artifacts aren't uploaded. Can't double-check that the shared framework bundles _lack_ Microsoft.AspNetCore.HeaderPropagation.dll until this is merged. That's also fine because we can look at what gets published after this is merged.